static/bidder-info/mabidder.yaml

```diff
@@ -0,0 +1,11 @@
+endpoint: "https://prebid.ecdrsvc.com/pbs"
+maintainer:
+  email: lmprebidadapter@loblaw.ca
```

Comment:
@ecdrsvc , PBS team attempted to verify above maintainer email. However PBS team was not able to reach you out on above specified email address.  

Note that an working maintainer email is needed to proceed with this PR

Reply:
sent an email again. Please reply with "received"